src/components/sidebar/SidebarNavigation/index.tsx

```diff
@@ -31,6 +32,19 @@ const isRouteEnabled = (route: string, chain?: ChainInfo) => {
   return !featureRoute || hasFeature(chain, featureRoute)
 }
 
+const NavItemLabel = ({ item }: { item: NavItem }) => {
+  const queueSize = useQueuedTxsLength()
+
+  if (item.href === AppRoutes.transactions.history) {
+    return <SidebarListItemCounter count={queueSize} />
+  }
+
+  if (item.href === AppRoutes.swap) {
+    return <Chip label="New" color="success" sx={{ ml: 1 }} />
+  }
```

Comment:
Should this be behind a feature flag or be a configuration constant somewhere? I don't know when we want to ermove it, but then we'll need to go in this file an delete this code.